cfme/tests/cli/test_evmserverd.py

```diff
@@ -38,6 +38,7 @@ def test_evmserverd_stop(appliance):
     })
 
     server_names = {server[server_name_key] for server in appliance.ssh_client.status["servers"]}
+    request.addfinalizer(lambda: appliance.ssh_client.run_command("systemctl start evmserverd"))
```

Comment:
https://github.com/ManageIQ/integration_tests/blob/master/cfme/utils/appliance/__init__.py#L1325
`<jkrocil> we shouldnt really do it through ssh_client directly`

Same with the other occurances.